src/components/DragAndDrop/index.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+
+import variables from '../../styles/variables';
+import DragAndDropPropTypes from './dragAndDropPropTypes';
+
+const COPY_DROP_EFFECT = 'copy';
+const NONE_DROP_EFFECT = 'none';
+
+const propTypes = {
+    ...DragAndDropPropTypes,
+
+    onDragOver: DragAndDropPropTypes.onDragOver,
```

Comment:
```suggestion
```

I think we can remove this.

Reply:
I am afraid we can not. 

Linter is not smart enough to see there is already definition in the dragAndDropPropTypes  for dragOver so it complains when i am trying to add defaultProps for dragOver

If you have any other solution for this problem, i am open for it

Reply:
Then It can't be helped. This is the easiest.